spec/models/service_provider_spec.rb

```diff
@@ -99,5 +99,15 @@
         expect(service_provider.ssl_certs.first.to_pem).to eq(pem)
       end
     end
+
+    context 'when a cert is named in the DB but does not exist on disk' do
```

Comment:
**question:** do we want to add a test for the case where one of the `certs` is an actual certificate (to test both branches)?

Reply:
the existing specs cover PEM content + filename that exists:

https://github.com/18F/identity-idp/blob/main/spec/models/service_provider_spec.rb#L83-L102